src/scenes/home/header/header.css

```diff
@@ -1,16 +1,21 @@
 .header {
   display: flex;
+  justify-content:space-around; 
```

Comment:
Don't forget a space between the attribute and the value! I also highly recommend setting up style lint.